app/controllers/concerns/inherited_proofing_concern.rb

```diff
@@ -28,34 +38,8 @@ def va_inherited_proofing_auth_code_params_key
     'inherited_proofing_auth'
   end
 
-  # Service Provider-agnostic members for now.
-  # Think about putting this in a factory(ies).
-
-  def inherited_proofing_service
-    inherited_proofing_service_class.new inherited_proofing_service_provider_data
-  end
-
-  def inherited_proofing_service_class
-    raise 'Inherited Proofing is not enabled' unless IdentityConfig.store.inherited_proofing_enabled
-
-    if va_inherited_proofing?
-      if IdentityConfig.store.va_inherited_proofing_mock_enabled
-        return Idv::InheritedProofing::Va::Mocks::Service
-      end
-      return Idv::InheritedProofing::Va::Service
-    end
-
-    raise 'Inherited proofing service class could not be identified'
-  end
-
-  def inherited_proofing_form(payload_hash)
-    return Idv::InheritedProofing::Va::Form.new payload_hash: payload_hash if va_inherited_proofing?
-
-    raise 'Inherited proofing form could not be identified'
-  end
-
   def inherited_proofing_service_provider_data
-    if va_inherited_proofing?
+    if inherited_proofing_service_provider == VA
```

Comment:
ditto, let's just check directly
```suggestion
    if inherited_proofing_service_provider == :va
```

Reply:
Done.